src/components/Input/Input.tsx

```diff
@@ -64,51 +83,33 @@ const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
         aria-invalid={error}
         {...htmlProps}
         sx={{
-          width: "100%",
-          paddingY: 2,
-          paddingX: 3,
-          paddingRight: unit ? 6 : 3,
-
-          backgroundColor: value ? "#F5F9FD" : "#F5F5F7", // TODO: remove fixed colors, use shade from theme
+          flexGrow: 1,
           outline: 0,
-          borderRadius: 2,
-          border: "2px solid transparent",
-
-          transition: "all 0.2s",
-          "&:focus, &:hover": {
-            borderColor: "highlight",
-            boxShadow: "0px 0px 0px 4px #EBF3FB", // TODO: remove fixed colors, use shade from theme
-            backgroundColor: "#FFFFFF", // TODO: remove fixed colors, use shade from theme
-          },
+          backgroundColor: "transparent",
+          border: "none",
+          p: 0,
 
-          // TODO: use typography styles
           color: "text",
-          fontWeight: 500,
-          fontSize: 14,
+          variant: "text.body1",
```

Comment:
Think it's not necessary anymore to use the body text styles after this commit https://github.com/micromedio/herz-ui/commit/c9457c30cac093ffcfb2b8b24ad7644b3596b91b, since the elements will inherit the root font styles https://herz-ui.micromed.io/#textfield

Reply:
Changed